File: week-02/day-3/sum.py
# - Write a function called `sum` that sum all the numbers
#   until the given parameter

number_1 = 5
number_2 = 10
number_3 = 3

def summery(*args):
    total = 0
    for a in args:
        total = total + a
    print(total)

summery(number_3 + number_2 + number_1)

File: week-07/day-3/sum.js
'use strict';
// - Write a function called `sum` that sum all the numbers until the given parameter
// - The function should return the result

function sumNumbers(number) {
    var total = 0;
    var digits = number.toString().split('');
    digits.forEach (function(num){
        total += parseFloat(num);
    });
    return total
}

console.log(sumNumbers(12345))
